contracts/FlowToken.cdc

```diff
@@ -259,28 +287,25 @@ pub contract FlowToken: FungibleToken, ViewResolver {
         // Example of how to resolve a metadata view for a Vault
         let ftView = vault.resolveView(Type<FungibleTokenMetadataViews.FTView>())
 
-        self.account.save(<-vault, to: /storage/flowTokenVault)
+        self.account.storage.save(<-vault, to: /storage/flowTokenVault)
 
         // Create a public capability to the stored Vault that only exposes
         // the `deposit` method through the `Receiver` interface
         //
-        self.account.link<&FlowToken.Vault{FungibleToken.Receiver, FungibleToken.Balance, MetadataViews.Resolver}>(
-            /public/flowTokenReceiver,
-            target: /storage/flowTokenVault
-        )
+        let receiverCapability = self.account.capabilities.storage.issue<&FlowToken.Vault>(/storage/flowTokenVault)
```

Comment:
with entitlements, we no longer need to restrict capabilities because now the privledged functions are hidden behind entitlements, so we can link the whole concrete type.

The syntax is invalid because cadence 1.0 remove restricted types, so you can't have a type specification that is `FlowToken.Vault{FungibleToken.Receiver}`. It has to either be `FlowToken.Vault` or `{FungibleToken.Receiver}`